src/metalog.c

```diff
@@ -267,6 +267,25 @@ static int parseLine(char * const line, ConfigBlock **cur_block,
             (void) 0;
         }
         else if (strcasecmp(keyword, "command") == 0) {
+            char *command = NULL;
+            struct stat st;
+            char *p = NULL;
+
+            if ((command = wstrdup(value)) == NULL) {
+                return -3;
+            }
+
+            p = strchr(command, (int) ' ');
+            if (p != NULL) {
+                *p = '\0';
+            }
+            if (stat(command, &st) < 0) {
+                doLog("Ignoring command \"%s\", because file doesn't exist", command);
```

Comment:
`stat` can fail for other reasons, and `strerror(errno)` could be used to provide the reason.

```c
doLog("Ignoring command \"%s\": %s", command, strerror(errno));
```

If not, it would be better to use "does not" instead of the contraction.

Reply:
Changed it, thanks!
Out of curiosity: What is the problem with such contractions? Can ' be a problematic character in logs? 
It seems, there already are some "n't"  in there. Should we also replace them?

Reply:
Contractions might confuse users who aren't fluent in English.